minimesos/src/main/java/com/containersol/minimesos/junit/MesosClusterTestRule.java

```diff
@@ -22,6 +25,15 @@
 
     private MesosCluster mesosCluster;
 
+    public static MesosClusterTestRule fromClassPath(String path) {
```

Comment:
It's nicer to load test files from the classpath. No need to refactor existing tests if we didn't have problems with this. Rather added it because this class is used by our users, so it's nice to give them the option to use best practices.